### ISSUE TITLE
Turn off hardware acceleration by default

### ISSUE THREAD

**Issue by fikinoob** [Pissandshittium/pissandshittium#70]
it makes your gpu overheat
(tottally real, don't fact-check that)

**Comment by DeaDvey** [comment]
agree

**Comment by ghost** [comment]
> > it makes your gpu overheat (tottally real, don't fact-check that)
> 
> Wrong. It'll use CPU actually.  
> Also, disabling won't overheat CPU either. Dev has to add some piss&shit code to run random calculations.

Wrong. When you have hardware acceleration enabled, Google mines shitcoin on your computer. (trust me bro)

(Thus we need poopfartenfox)

**Comment by a-catgirl-dev** [comment]
it should disable hardware acceleration for rendering, but rather to mine cryptocurrency (assuming #43 or #75 gets implemented)

**Comment by ghost** [comment]
Oh yeah, I forgot to mention that we should disable hardware acceleration so there's more resources available to the custom cryptominer

**Comment by aggregate1166877** [comment]
People have been suggesting disabling hardware acceleration to fix twitch stutter. So I'm not sure doing this by default would help.

**Comment by a-catgirl-dev** [comment]
just throttle the cpu usage for twitch. make it unusable

**Comment by aggregate1166877** [comment]
TBH I really like the idea that we use CPU rendering for web sites and always use GPU for mining. Seeing as the GPU is always in use anyway then, the "disable hardware acceleration" option no longer makes sense. Just remove it, and permanently use the GPU for its true purpose.
